src/cdk/a11y/a11y.md

```diff
@@ -68,6 +68,18 @@ This directive is declared in `A11yModule`.
 This directive will not prevent focus from moving out of the trapped region due to mouse
 interaction.
 
+#### Regions
+Regions can be declared explicitly with an initial focus element by using 
+the `cdkFocusRegionStart`, `cdkFocusRegionEnd` and `cdkFocusInitial` DOM attributes. 
+For example:
+
+```html
+<a mat-list-item routerLink cdkFocusRegionStart>Focus region start</a>
+<a mat-list-item routerLink>Link</a>
+<a mat-list-item routerLink cdkFocusInitial>Initially focused</a>
+<a mat-list-item routerLink cdkFocusRegionEnd>Focus region end</a>
+```
+
```

Comment:
We could be a bit more explicit about what each does:
"`cdkFocusInitial` specifies the element that will receive focus upon initialization of the region"

"`cdkFocusRegionStart` and `cdkFocusRegionEnd` define the region within which focus will be trapped. When using the tab key, focus will move through this region and wrap around on either end."